packages/xo-server/src/xapi/index.js

```diff
@@ -2114,6 +2108,8 @@ export default class Xapi extends XapiBase {
     const vifRef = await this.call(
       'VIF.create',
       filterUndefineds({
+        currently_attached:
```

Comment:
default value is `true` in this method whereas the default value in xapi is `false`